### ISSUE TITLE
Unable to start "Getting started" from a Yarn workspace

### ISSUE THREAD

**Issue by romain-ortega** [nuxt/framework#8802]
### Environment

- Operating System: `Darwin`
- Node Version:     `v16.14.0`
- Nuxt Version:     `3.0.0-rc.13`
- Nitro Version:    `0.6.1`
- Package Manager:  `yarn@3.2.0`
- Builder:          `vite`
- User Config:      `-`
- Runtime Modules:  `-`
- Build Modules:    `-`

### Reproduction

From within `/apps/admin-nuxt`
```
npx nuxi create admin-nuxt
cd my-app
yarn install
yarn dev -o
```

### Describe the bug

<img width="836" alt="image" src="https://user-images.githubusercontent.com/17955572/200539926-067790dd-39f1-46d2-a0de-f52f7c19077c.png">

### For Additional context

`ls: ../../node_modules/@nuxt/vite-builder: No such file or directory`

```
$ yarn why @nuxt/vite-builder
└─ nuxt@npm:3.0.0-rc.13
   └─ @nuxt/vite-builder@npm:3.0.0-rc.13 [bd2df] (via npm:3.0.0-rc.13 [bd2df])

```

If I execute the following command: 
`yarn add @nuxt/vite-builder`

Then I have theses errors, when starting the app (`yarn dev -o`) :

<img width="837" alt="image" src="https://user-images.githubusercontent.com/17955572/200540671-abada4bf-4892-445e-bdff-e17b79b6b8d8.png">

**Comment by danielroe** [comment]
Nuxt does not yet support yarn pnp. You can use nodeModules linker. Despite the name, we are tracking both pnpm without shamefully-hoist + yarn pnp in https://github.com/nuxt/framework/issues/5411.

**Comment by romain-ortega** [comment]
@danielroe We're not using pnp, this is our root `.yarnrc.yml` :
`nodeLinker: node-modules`

**Comment by danielroe** [comment]
I can't reproduce with the following steps:

```js
npx nuxi init yarn-app
cd yarn-app
yarn set version stable
echo 'nodeLinker: node-modules' >> .yarnrc.yml
yarn install
yarn dev
```

That makes me think the issue is your yarn setup.

**Comment by romain-ortega** [comment]
I have created a reproduction repository here : 
https://github.com/romain-ortega/nuxt3-reproduction

**Comment by romain-ortega** [comment]
Solved with latest stable release (v3.0.0)
